src/pages/ImportData.js

```diff
@@ -63,15 +90,73 @@ const ImportData = () => {
     div({ role: 'main', style: styles.container }, [
       div({ style: styles.card }, [
         div({ style: styles.title }, ['Importing Data']),
-        div({ style: { fontSize: 16 } }, ['From: ', new URL(url).hostname])
+        div({ style: { fontSize: 16 } }, ['From: ', new URL(url).hostname]),
+        div({ style: { marginTop: '1rem' } }, [
+          'The dataset(s) you just chose to import to Terra will be made available to you within a workspace of your choice where you can then perform analysis.'
+        ]),
+        div({ style: { marginTop: '0.5rem' } }, [
+          'You can import into a new blank workspace or an existing workspace you have already been working on.'
+        ])
       ]),
       div({ style: { ...styles.card, marginLeft: '2rem' } }, [
-        div({ style: styles.title }, ['Destination Workspace']),
-        h(WorkspaceImporter, {
-          authorizationDomain: ad,
-          selectedWorkspaceId: wid,
-          onImport
-        }),
+        Utils.switchCase(mode,
+          ['existing', () => {
+            return h(Fragment, [
+              div({ style: styles.title }, ['Start with an existing workspace']),
+              h(WorkspaceSelector, {
+                workspaces: _.filter(ws => {
+                  return Utils.canWrite(ws.accessLevel) &&
+                    (!ad || _.some({ membersGroupName: ad }, ws.workspace.authorizationDomain))
+                }, workspaces),
+                value: selectedWorkspaceId,
+                onChange: setSelectedWorkspaceId
+              }),
+              div({ style: { display: 'flex', alignItems: 'center', marginTop: '1rem' } }, [
+                h(ButtonPrimary, {
+                  disabled: !selectedWorkspace,
+                  onClick: () => onImport(selectedWorkspace.workspace)
+                }, ['Import']),
+                h(ButtonSecondary, { style: { marginLeft: '1rem' }, onClick: () => setMode() }, ['Back'])
+              ])
+            ])
+          }],
+          ['new', () => {
+            return h(Fragment, [
+              div({ style: styles.title }, ['Start with a new workspace']),
```

Comment:
Nit: I find it a little jarring/distracting for this pane to change at the same time as opening a modal. Part of it is that the modal partially obscures the new content so I feel like I missed something.